### ISSUE TITLE
`[build] headless` config option builds an app in a headful mode

### ISSUE THREAD

**Issue by chicoxyzzy** [socketsupply/socket#514]
This is a regression. Note that `ssc build --headless` works in the CLI.

**Comment by chicoxyzzy** [comment]
It works with `.sscrc`:
```ini
[build]
headless = true
``

**Comment by chicoxyzzy** [comment]
fixes

**Comment by lamiazar** [comment]
I tested multiple test cases.
Sergey is working on fixing:
```
socket.ini
headless = true

.sscrc
headless = false
```
which is not working as expected.

**Comment by chicoxyzzy** [comment]
fixed in #524
